.github/workflows/testWlsVmAdmin.yml

```diff
@@ -45,8 +49,25 @@ env:
 
 jobs:
   preflight:
+    outputs:
+      enableELK: ${{ steps.setup-env-variables-based-on-dispatch-event.outputs.enableELK }}
     runs-on: ubuntu-latest
     steps:
+      - name: Setup environment variables
+        id: setup-env-variables-based-on-dispatch-event
+        run: |
+          ref='refs/heads/main'
+          if [ ${{ github.event_name }} == 'workflow_dispatch' ]; then
+            enableELK=${{ github.event.inputs.enableELK }}
+            ref=${{ github.event.inputs.ref }}
+          else
+            enableELK=${{ github.event.client_payload.enableELK }}
+            ref=${{ github.event.client_payload.ref }}
```

Comment:
My same comments regarding the corresponding `buildWlsVm4*` files applies here.  Also, I judge that these changes need to also be made to 

```
testWlsVmCluster.yml
testWlsVmDynamicCluster.yml
```

Reply:
Yes, will get to them in different PRs. After this PR gets in, I will create issues for other offer repos